python/py-spiceypy/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        AndrewAnnex SpiceyPy 3.1.1 v
+
+checksums           rmd160  eb48ae08ee964c659bb61a4bcfc8ce6db8169e74 \
+                    sha256  6281e703fd7c8c8a894c0693ac7b336f26b595c1a2c54666d8093d215d07211e \
+                    size    350722
+
+name                py-spiceypy
+platforms           darwin
+maintainers         nomaintainer
+license             MIT
+description         SpiceyPy: The NASA JPL NAIF SPICE toolkit wrapper for Python
+long_description    ${description}
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-certifi \
+                    port:py${python.version}-numpy \
```

Comment:
`py-numpy` is also needed at runtime, so that one should be added as `depends_lib-append`; I am not sure yet about `py-certifi` but will try to build it locally and see what happens

Reply:
> py-numpy is also needed at runtime

Indeed.

Reply:
`numpy` should be removed from `depends_build`; `depends_lib` implies already that a dependency is need at build- and run-time.